assets/javascripts/discourse/initializers/preview-edits.js.es6

```diff
@@ -39,7 +39,6 @@ export default {
 
       setupListStyle: function() {
         if (this.get('socialMediaStyle')) {
-          this.set('skipHeader', true)
```

Comment:
@pmusaraj Please make this contingent on your new setting being false rather than removing it.